docs-chef-io/content/workstation/getting_started.md

```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
     weight = 40
 +++
 
-This guide contains common configuration options used when setting up a
-new Chef Workstation installation. If you do not have Chef Workstation
-installed, see its [installation guide](/workstation/install_workstation/)
-before proceeding further.
+This guide walks your through the four parts to set up Chef Workstation on your computer.
 
-## Configure Ruby Environment
+* [Setup Ruby for Chef Workstation]({{< relref "#Setup Ruby for Chef Workstation" >}})
+* [Set up your chef-repo]({{< relref "#Setup Your Chef Repo" >}}) for storing your cookbooks
+* [Understand your Chef Workstation installation]({{< relref "#Your Chef Workstation Installation" >}})
+* [Set Up Chef Infra Communication]({{< relref "#Set Up Chef Infra Communications" >}})
 
-For many users of Chef, the version of Ruby that is included in Chef
-Workstation should be configured as the default version of Ruby on your
-system.
+In order to use Chef Workstation effectively, you need to install and set up [Chef Infra Server]({{< relref "/server/install_server.md" >}}) first.
 
-{{< note >}}
+If you do not have Chef Workstation installed, start with the [installation guide]({{< relref "/workstation/install_workstation.md" >}}).
 
-These instructions are intended for macOS and Linux users. On Windows Chef Workstation includes a desktop shortcut to a PowerShell prompt already configured for use.
+## Prerequisites
 
-{{< /note >}}
+1. [Download and install Chef Workstation]({{< relref "install_workstation.md" >}})
+1. A running instance of [Chef Infra Server]({{< relref "server/install_server.md" >}}) or [Hosted Chef Server](https://manage.chef.io/signup) and access to the:
+   1. `ORGANIZATION-validator.pem`
+   1. `USER.pem`
+1. Install a [Code Editor]({{< relref "#Code Editors" >}}) (optional)
 
-1. Open a terminal and enter the following:
+## Setup Ruby for Chef Workstation
+
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
+
+* If you don't plan on writing a lot of Chef cookbooks and you're not a Ruby developer, then we recommend that you use the [Chef Workstation Ruby as the system default]({{< relref "#Use Chef-Workstation Ruby as the System Default">}})
+* If you are a Ruby developer, then set up Ruby with an [environment variable]({{< relref "#Add Ruby as an Environment Variable" >}}).
```

Comment:
`Add Ruby as an Environment Variable` doesn't really make sense as a way to describe this. It should be adding ruby to the system PATH